composables/api/useConfirmAccount/index.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiResponse, optionalParameters, useApi } from 'composables/useApi'
+import { Account } from 'lib/api/Account'
+
+export const useConfirmAccount = async (
```

Comment:
same as above comment, I really feel like Function Declerationsn are a better pattern in most cases, and should be preferred over Function Expressions.